learn/tanssi/tanssi-ethereum-bridge.md

```diff
@@ -0,0 +1,254 @@
+
+### Tanssi to Ethereum Consensus  { : #tanssi-ethereum-consensus }
+
+The BEEFY (Bridge Efficiency Enabling Finality Yielder) consensus protocol operates as a crucial prover from the Tanssi side of the bridge. It was specifically designed to facilitate efficient and trustless bridging to external blockchains like Ethereum that were not natively built with interoperability features in mind.
+
+```mermaid
+sequenceDiagram
+    %%{init: {'sequence': {'mirrorActors': false}}}%%
+    participant Tanssi_Module as Tanssi <br/> BEEFY Module (prover)
+    participant Beefy_Relayer as Relayer <br/> (Beefy)
+    participant Eth_BeefyClient as Ethereum <br/> BEEFY Client (verifier)
+
+    Tanssi_Module->>Beefy_Relayer: Generate BEEFY Commitment
+    activate Beefy_Relayer
+    Beefy_Relayer->>Eth_BeefyClient: Submit commitment/proof
+    deactivate Beefy_Relayer
+
+    activate Eth_BeefyClient
+    Eth_BeefyClient->>Eth_BeefyClient: Verify commitment
+    deactivate Eth_BeefyClient
+
+```
+
+### Ethereum to Tanssi Consensus { : #ethereum-tanssi-consensus }
+
+When bridging from Ethereum to Tanssi, the Ethereum Beacon Chain's consensus mechanism acts as the prover. The core idea is to provide Tanssi's on-chain light client with undeniable proof of Ethereum's finalized state, including specific events or messages destined for Tanssi.
+
+```mermaid
+sequenceDiagram
+    %%{init: {'sequence': {'mirrorActors': false}}}%%
+    participant Eth_BeaconCons as Ethereum <br/> Beacon Chain Consensus(Prover)
+    participant Beacon_Relayer as Relayer <br/> (Beacon)
+    participant Tanssi_EthClient as Tanssi <br/> Ethereum Light Client (verifier)
+
+    Eth_BeaconCons->>Beacon_Relayer: Beacon chain update (Header/Proof)
+    activate Beacon_Relayer
+    Beacon_Relayer->>Tanssi_EthClient: Submit update/proof
+    deactivate Beacon_Relayer
+
+    activate Tanssi_EthClient
+    Tanssi_EthClient->>Tanssi_EthClient: Verify update/proof
+    deactivate Tanssi_EthClient
+```
+
+Viewing the bridge from a messaging perspective reveals how it manages cross-chain communication, leveraging the underlying consensus verification layer for security. Dedicated relayers are responsible for message transport: the Execution relay handles transfers from Ethereum to Tanssi, and the Tanssi relay manages the flow from Tanssi to Ethereum.
```

Comment:
Why is Execution capitalized here? I might be missing something, so mostly asking out of curiosity